dev/com.ibm.ws.kernel.boot.core/src/com/ibm/ws/kernel/boot/internal/commands/PackageProcessor.java

```diff
@@ -170,6 +172,13 @@ protected File buildManifestForIncludeHasRunnable(File installationManifest) thr
         mf.getMainAttributes().putValue("Main-Class", "wlp.lib.extract.SelfExtractRun");
         mf.getMainAttributes().putValue("Server-Name", processName);
 
+        // For Java 9, we need to apply the /wlp/lib/platform/java/java9.options to the manifest.
```

Comment:
the args in java9.options are things we need to run all features available in liberty, but it is likely that the SelfExractRun just needs a subset of these because all it's doing is launching a separate JVM to do the extract and then it exits (I think).

So instead of copying things over from java9.options to the manifest at runtime, can we just add the flags necessary to do a clean SelfExtract to the jar's manifest directly?